### PR TITLE
Add codepush command tests

### PR DESCRIPTION
**Pull request by n-kurdiukov-akvelon**
Added codepush command tests
Fixed some error handling issues in CLI code

### CODE REVIEW

**Review by snowpardx — APPROVED** [review]

**Review by ruslan-bikkinin — APPROVED** [review]

File: src/commands/codepush/deployment/rename.ts [hunk 1]
```diff
@@ -48,10 +48,10 @@ export default class CodePushRenameDeploymentCommand extends AppCommand {
     } catch (error) {
       debug(`Failed to rename deployments - ${inspect(error)}`);
       if (error.statusCode === 404) {
-        const appNotFoundErrorMsg = `The deployemnt ${this.currentDeploymentName} for app ${this.identifier} does not exist.`;
+        const appNotFoundErrorMsg = `The deployment ${this.currentDeploymentName} for app ${this.identifier} does not exist.`;
         return failure(ErrorCodes.NotFound, appNotFoundErrorMsg);
-      } else if ((error.statusCode = 409)) {
```
Comment (anywherepilot) [inline src/commands/codepush/deployment/rename.ts:53]: That's a nasty one. There should be a warning for this pattern =/

**Review by anywherepilot — COMMENTED** [review]
This looks like it was a serious amount of work. Good to see codepush getting some love in the CLI. Many of the tests fail though. You'll have to get them all to pass in the CI setup.

**Review by andreidubov — DISMISSED** [review]

**Review by snowpardx — APPROVED** [review]

**Pull request merged**